### ISSUE TITLE
Interface from same package name as dependency broken since v3.2.1

### ISSUE THREAD

**Issue by jeanschmitt** [gojuno/minimock#87]
Hello,

Since v3.2.1, minimock generates import groups that don't compile, for interfaces importing types from other packages with the same name as the interface's package.

# Steps to reproduce

Consider this directory structure:

```
├── pb
│   └── interface.go
├── dependency
│   └── pb
│       └── dep.go
└── gen.go
```

With the following contents:

```go
// dependency/pb/dep.go
package pb

type Item struct {
	A string
}
```

```go
// pb/interface.go
package pb

import (
	"mymodule/dependency/pb"
)

type Interface interface {
	A() pb.Item
}
```

```go
// gen.go
package mymodule

//go:generate minimock -i ./pb.Interface -g
```

## Generated import group in v3.2.0

```go
import (
	"mymodule/dependency/pb"
	mm_atomic "sync/atomic"
	mm_time "time"

	"github.com/gojuno/minimock/v3"
)
```

## Generated import group in v3.2.1

```go
import (
	"mymodule/pb"

	"mymodule/dependency/pb"
	mm_atomic "sync/atomic"
	mm_time "time"

	"github.com/gojuno/minimock/v3"
)
```

Which raises the error `'pb' redeclared as the imported package name`

**Comment by zcolleen** [comment]
Hey @jeanschmitt thanks for issue! I started another issue in gowrap https://github.com/hexdigest/gowrap/issues/87 as this bug is connected with gowrap update.

**Comment by nordicdyno** [comment]
maybe it's worth to rollback gowrap in minimock dependencies until it will be fixed? because it's update blocker for us

**Comment by zcolleen** [comment]
fix is available at v3.4.0
